central_v2/src/components/CardCarousal.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { APIClients, IGameTemplate } from '@righton/networking';
+import { Swiper, SwiperSlide, SwiperRef } from 'swiper/react';
+import { Pagination } from 'swiper/modules';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import { useTheme } from '@mui/material/styles';
+import StyledGameCard from './GameCard';
+import placeHolder from '../images/placeHolder.svg';
+
+
+interface GameCardCarouselProps {
+    apiClients: APIClients;
+    recommendedGames: IGameTemplate[];
+}
+
+export default function GameCardCarousel({ apiClients, recommendedGames }: GameCardCarouselProps) {
+    const theme = useTheme();
+    const swiperRef = useRef<SwiperRef>(null);
+
+
+    return (
+        <Swiper
+            style={{ width: '100%' }}
+            modules={[Pagination]}
+            pagination={{
+                el: '.swiper-pagination-container',
+                bulletClass: 'swiper-pagination-bullet',
+                bulletActiveClass: 'swiper-pagination-bullet-active',
+                clickable: true,
+                renderBullet(index: number, className: string) {
+                    return `<span class="${className}" style="width:20px; height:6px; border-radius:2px"></span>`;
+                },
+            }}
+            ref={swiperRef}
+            spaceBetween={theme.sizing.smPadding}
+            centeredSlides
+            loop
+            navigation
+            breakpoints={{
+                '375': {
```

Comment:
How'd we arrive at these `slidesPerVew` numbers? Not complaining if they work but they seem super precise